readme.md

```diff
@@ -325,6 +325,19 @@ A `streamName` can be passed to pipe `"stderr"`, `"all"` (both `stdout` and `std
 
 Returns `execaChildProcess`, which allows chaining `.pipe()` then `await`ing the [final result](#childprocessresult).
 
+#### kill(signalOrError?)
+
+`signalOrError`: `string | number | Error`\
+_Returns_: `boolean`
```

Comment:
I am not sure which convention to use to document the return value, please let me know if this works.

Reply:
Yeah, that's fine.